apps/pie-docs/src/assets/styles/base/_shared.scss

```diff
@@ -0,0 +1,110 @@
+@use '../fozzie-config' as f;
+@use '../variables' as v;
+
+.c-content {
```

Comment:
does `_shared.scss` make sense for this file?

Reply:
could rename to `sharedContent` - it's currently for css styles that are shared between the header and content page 👍

Reply:
Instead I've added these styles to the `layout.scss` file with a note explaining that the styles refer to the layout of both the homePage and contentPage  👍